### PR TITLE
Readme improvements

### PR DESCRIPTION
**Pull request by sabre1041**
Revamped the README

### CODE REVIEW

**Comment by usrbinkat** [comment]
A few suggested tweaks supplied in PR https://github.com/sabre1041/uor-client-go/pull/1

File: README.md [hunk 1]
```diff
@@ -47,321 +47,555 @@ uor-client-go version
...
 ### Pull subsets of a UOR collection to a location by attribute
 
-```
+Pull a portion of a collection by filtering for a set of attribute:
+
+```shell
 uor-client-go pull localhost:5000/myartifacts:latest -o my-output-directory --attributes key=value
 ```
 
 ## Getting Started
```
Comment (usrbinkat) [inline README.md:99]: We should probably plan on moving technical information into the docs/ path pending resolution of https://github.com/uor-framework/universalreference.io/issues/17

**Review by usrbinkat — APPROVED** [review]
It is very exciting to have all this to show off so early in the project.

Making this information accessible is a very high priority so let's get this merged in now.

In the very near future we should consider relocating the heavily technical information into actual docs.

File: README.md (hunk 1 above)
Comment (usrbinkat) [reply README.md:99]: Relevant to suggestion https://github.com/uor-framework/universalreference.io/pull/22

**Comment by sabre1041** [comment]
Hold off on merging this PR. I am going to make a number of modifications

**Comment by usrbinkat** [comment]
> Hold off on merging this PR. I am going to make a number of modifications

I'd like to merge the docs tooling pr's before this. Then we can move some of this content into docs properly.

How does that fit with your plans?

**Comment by sabre1041** [comment]
> > Hold off on merging this PR. I am going to make a number of modifications
> 
> I'd like to merge the docs tooling pr's before this. Then we can move some of this content into docs properly.
> 
> How does that fit with your plans?

The key enhancement that I was going to circle back on centers around the local environment that most users would use as part of the getting started (running a vanilla `registry:2` registry. This means that, by default, the registry would not be secured with TLS certificates and consequently, most of the commands would require the `--plain-http` flag.

I was going to add a "Setting up your environment" section and then modify the relevant commands to use the `--plain-http` flag.

I would most likely complete this by Wednesday Sep 7 at the latest

**Comment by sabre1041** [comment]
@usrbinkat Updated the README with the desired changes. Feel free to take a look

**Comment by usrbinkat** [comment]
LGTM Thanks @sabre1041!

File: README.md [hunk 2]
```diff
@@ -47,321 +47,599 @@ uor-client-go version
...
 uor-client-go pull localhost:5000/myartifacts:latest -o my-output-directory
 ```
 
 ### Pull subsets of a UOR collection to a location by attribute
 
-```
+Pull a portion of a collection by filtering for a set of attribute:
+
+```shell
 uor-client-go pull localhost:5000/myartifacts:latest -o my-output-directory --attributes key=value
```
Comment (jpower432) [inline README.md:96]: I know you didn't change this line, but I noticed it still shows `key=value` when it should be an `attribute-query.yaml`.

**Review by jpower432 — COMMENTED** [review]
This is a great change. I only have one comment about an issue I noticed in the README.md in general, but not done by the changes in this PR.

**Review by jpower432 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (sabre1041) [reply README.md]: @jpower432 [fd2dce0](https://github.com/uor-framework/uor-client-go/pull/85/commits/fd2dce07d2c1e6d47755bbff790df0713b9a1a75) contains an update. Thanks for noting it!

**Pull request merged**